pkg/networkservice/common/clientinfo/client.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
Looks like you may need to rebase now that the other PR has been merged... I think these files are from the last PR :)